utils/update_build_version.py

```diff
@@ -71,21 +89,32 @@ def describe(dir):
             return 'unknown hash, ' + datetime.date.today().isoformat()
 
 
+def getVersionString(project, dir):
```

Comment:
Hm.  I see the style guide says this.
In that case we should change deduceSoftwareVersion should be changed too.
(And do the same in SPIRV-Tools in another CL).